tx-pool/src/component/proposed.rs

```diff
@@ -300,22 +306,24 @@ mod tests {
         let tx1_hash = tx1.hash();
         let tx2 = build_tx(vec![(&tx1_hash, 0)], 1);
 
-        let mut pool = ProposedPool::new();
+        let mut pool = ProposedPool::new(DEFAULT_MAX_ANCESTORS_SIZE);
 
         pool.add_entry(TxEntry::new(
             tx1.clone(),
             MOCK_CYCLES,
             MOCK_FEE,
             MOCK_SIZE,
             get_related_dep_out_points(&tx1, |_| None).unwrap(),
-        ));
+        ))
+        .unwrap();
```

Comment:
There's a lot of unwrap added here, any chance we can properly deal with the errors? Or are we sure they won't happen?

Reply:
This is test code.